addons/web/static/src/views/sample_server.js

```diff
@@ -340,6 +342,34 @@ export class SampleServer {
     _getRandomSubRecordId() {
         return Math.floor(Math.random() * SampleServer.SUB_RECORDSET_SIZE) + 1;
     }
+
+    /**
+     * Simulate a 'name_get' operation
+     *
+     * @private
+     * @param {Object} params
+     * @param {string} params.model
+     * @param {Array[]} params.args
+     * @returns {Array[]} a list of [id, display_name]
+     */
+    _mockNameGet(params) {
+        const { model, args } = params;
+        let ids = args[0];
+        if (!args.length) {
+            throw new Error("name_get: expected one argument");
+        } else if (!ids) {
+            return [];
+        }
+        if (!Array.isArray(ids)) {
+            ids = [ids];
+        }
+        const { records } = this.data[model];
+        const names = ids.map((id) =>
+            id ? [id, records.find((r) => r.id === id).display_name] : [null, ""]
```

Comment:
[null, ""] ? it's weird to handle such a case, not sure the real server does that. We should never ask a name_get of a falsy id anyway.